### ISSUE TITLE
Message box doesn't receive keyboard focus properly on open

### ISSUE THREAD

**Issue by peter-b** [livecode/livecode-ide#591]
This is a weird one, and might be a (platform-specific) engine bug rather than an IDE bug!  This affects the IDE when running on Linux using the KWin (KDE) window manager.

When the message box is opened by clicking the "Message Box" button on the menu bar, it partially receives the focus.  The message box text is selected, and typing overwrites/modifies the contents of the text. However, the window manager focus remains on the menu bar (you can tell because of the highlighting of the window decorations) and pressing <Return> to run the command doesn't work. To run the command, you have to change the window focus to the message box using the mouse.

**Comment by peter-b** [comment]
This is the same as [bug 15610](http://quality.livecode.com/show_bug.cgi?id=15610), probably.
